### DISCUSSION TITLE
Symfony\Mailer (SES Bridge) defining profiles and missing/bad docs

### DISCUSSION THREAD

**Discussion by oojacoboo** [symfony/symfony#51373] [Q&A]
### Symfony version(s) affected

6.2.0

### Description

Firstly, the docs for the Symfony AWS SES bridge don't include much at all, outside of embedding the access key and secret into the DSN.  And doing so is not an ideal way of managing AWS keys.  Even the `~/.aws/credentials` is seen as less ideal to newer authentication methods AWS is pushing.

What does `default` even mean in `ses+api://default?region=us-east-1`?  Is that the name of the profile (don't think so - transport factory calls it a "host")?  Replacing that with anything else results in a host resolution error for `https://whatever/v2/email/outbound-emails`.  I have no idea what host that is (maybe some internal SES host), and if that's `default` what kind of host is that?

If a dev has a `[default]` profile configured in their `~/.aws/credentials` config with adequate permissions for their IAM user/role, things seem to work properly.  But what happens when a dev needs to manage multiple AWS accounts with unique profile identifiers?  How do you tell the DSN which profile to use?

### How to reproduce

Try using any AWS profile other than `default`.

### Possible Solution

Looking at the `SesTransportFactory` and the `AsyncAws\Core\Configuration` it looks like a `profile` could be passed in the `Configuration::create` call if the DSN parsed it.  The use of constants in this AWS `Configuration` class is quite strange, making it difficult to easily tell how it's working.

I guess what really needs to be done is to add additional query params to the DSN to support additional configuration options for the `SesClient` - especially for `profile`.

**Comment by derrabus** [comment]
If you think that the documentation needs to be improved, please open an issue at https://github.com/symfony/symfony-docs/. Because you're mainly asking a lot of questions in your "bug report", I'm moving this issue to support.
